submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="app.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <title>Popup</title>
+  </head>
+  <body>
+    <nav>
+      <div class="nav-container">
+        <div class="nav-el nav-el-text" tabindex="0">
+          <p>Mail</p>
+        </div>
+        <div class="nav-el nav-el-text" tabindex="0">
+          <p>Images</p>
+        </div>
+
+        <!-- #wrap-el - обертка для input + label + popup.
+              CSS к #wrap-el не применял -->
+        <div id="wrap-el">
+
+          <input type="checkbox" id="button-1" tabindex="0">
+          <div class="nav-el" id="nal-el-menu">
+            <label for="button-1" id="menu-label">
+                <img src="img/apps.jpg" alt="Menu icon" />
+            </label>
+          </div>
+<!-- закончил рефакторинг тут -->
+          <section id="apps-window">
+            <div id="top-triangle"></div>
```

Comment:
Also such "styles elements" better to replace by `css` `:before` or `:after` ([more info](https://developer.mozilla.org/ru/docs/Web/CSS/::before)).